benchmarks/distributed/rpc/rl/observer.py

```diff
@@ -31,14 +31,17 @@ def run_ob_episode(self, agent_rref, n_steps):
         state, ep_reward = self.reset(), None
         rewards = torch.zeros(n_steps)
         observer_latencies = []
+        agent_latencies = []
 
         for st in range(n_steps):
             ob_latency_start = time.time()
             # action = agent_rref.rpc_sync().select_action(self.id, state)
             action = rpc_sync(agent_rref.owner(), self.select_action, args=(
                 agent_rref, self.id, state))
+
             observer_latencies.append(time.time() - ob_latency_start)
+
```

Comment:
@mrshenli can you please confirm observer latency is being calculated correctly

Reply:
Looks correct to me. Since the `step` function below is not part of the measurements that we are interested in, it should be fine to leave it out.